_lifeofinternatpuppet/post_7.md

```diff
@@ -41,17 +41,43 @@ On Tuesday 13th October, after finishing the session with Sheena and Ciaran, I w
 
 Why not check this [link out](https://puppet.com/blog/litmus-new-module-acceptance-testing-tool/) to find out more information about Litmus.
 
-The second part of the session with Daiana was on testing a pull request (pr) from the community, my first ticket was assigned to me (changing author name of supported modules to puppetlabs). A to-do-list was given by Daiana on what I learnt in part 2 of the meeting. She also suggested to keep a word document on all the necessary commands used on a day-to-day basis when doing work, She felt having that document really reduces the stress to search for commands within the MAC OS terminal when working on a ticket/issue.
-
-I still have to put together a document which will include all the necessary... stay tuned for it!
+The second part of the session with Daiana was on testing a pull request (pr) from the community, my first ticket was assigned to me (changing author name of supported modules to puppetlabs). A to-do-list was given by Daiana on what I learnt in part 2 of the meeting. Check out what I was working on in the to do list given below:-
+
+Part 1:
+1. Choose a PR from PR reviews report from Community
+2. Clone it locally
+3. provision (choose what you want from provision list)
+4. see if inventory file was created
+5. cat inventory file
+6. install agent
+7. install module
+8. run tests
+9. tear down
+Part 2:
+1. Choose one vm and provision (only that vm)
+2. install agent
+3. install module
+4. run tests
+5. tear down
+Part 3:
+1. Use target host to run a specific test
+Part 4.
+on the same PR from previous points
+1. provision with release checks 5
+2. install puppet 5 (agent)
+3. install module
+4. run tests
+5. tear down
```

Comment:
Also, here, if you don't have the technical details added somewhere else , I would add them for each point just to provide more skills for you (as this is for your Uni)